MediaManager/IMediaManager.cs

```diff
@@ -50,6 +51,14 @@ public interface IMediaManager : IPlaybackManager, IDisposable
         /// <returns></returns>
         Task<IMediaItem> Play(string uri);
 
+        /// <summary>
+        /// Plays an embeded resource
+        /// </summary>
+        /// <param name="resourceName"></param>
+        /// <param name="assembly"></param>
+        /// <returns></returns>
+        Task<IMediaItem> Play(string resourceName, Assembly assembly);
```

Comment:
Assembly assembly = null

Reply:
Added `TryFindAssembly` in `MediaExtractorBase`